neural_compressor/config.py

```diff
@@ -1343,7 +1355,37 @@ def __init__(self):
         pass
 
 class Torch2ONNXConfig(ExportConfig):
-    """Config Class for Torch2ONNX."""
+    """Config Class for Torch2ONNX.
+
+    Args:
+        dtype (str, optional): The data type of the exported model, select from ["fp32", "int8"]. 
+                               Defaults to "int8".
+        opset_version (int, optional): The ONNX opset version used for export. Defaults to 14.
+        quant_format (str, optional): The quantization format of the exported int8 onnx model, 
+                                      select from ["QDQ", "QLinear"]. Defaults to "QDQ".
+        example_inputs (required): Example inputs used for tracing model. Defaults to None.
```

Comment:
type for example_inputs?

Reply:
it has lots of kinds, like dict, tuple, list, tensor, I cannot specify it.

Reply:
We should list the types that we accept.